app/serializers/shipit/tail_task_serializer.rb

```diff
@@ -33,7 +33,7 @@ def include_rollback_url?
     delegate :stack, to: :object
 
     def offset
-      (output&.bytesize || 0) + (context[:last_byte] || 0)
+      (output&.bytesize || 0) + (context[:last_byte]&.to_i || 0)
```

Comment:
that ` || 0` is no longer useful then

Reply:
There is still a chance that `context[:last_byte]` is `nil`. In that case we do need the `|| 0`, otherwise we end up with a "No implicit conversion of `nil` to integer" error.

Reply:
Either way I think the proper fix would have bene to do the casting in the controller. I might push a tiny refactor on master.

Reply:
c25a37c7678cf0a9250e4cd9b15211ed779d5c48